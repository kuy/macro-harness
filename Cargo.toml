[package]
name = "macro-harness"
version = "0.0.1"
authors = ["Yuki Kodama<endflow.net@gmail.com>"]
repository = "https://github.com/kuy/macro-harness"
homepage = "https://github.com/kuy/macro-harness"
license = "MIT"
edition = "2018"

[dependencies]
anyhow = "1.0"
cargo_toml = "0.11.5"
console = "0.15"
rand = "0.8.5"
similar = { version = "2.2", features = ["inline"] }
toml = "0.8.12"

[dev-dependencies]
similar-asserts = "1.4.2"
