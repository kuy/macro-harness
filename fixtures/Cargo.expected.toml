[package]
name = "awesome-macro-tests"
edition = "2018"
version = "0.0.0"
authors = []
keywords = []
categories = []
autobins = true
autoexamples = true
autotests = true
autobenches = true
publish = false
[dependencies.awesome]
path = "/home/macro-harness/projects/awesome"
features = []
optional = false

[dependencies.awesome-macro]
path = "/home/macro-harness/projects/awesome-macro"
features = []
optional = false

[dev-dependencies]

[build-dependencies]

[target]

[features]

[patch]

[lib]
path = "/home/macro-harness/projects/awesome/tests/test_macro.rs"
test = true
doctest = true
bench = true
doc = true
plugin = false
proc-macro = false
harness = true
edition = "2018"
required-features = []

[profile]
[badges.maintenance]
status = "none"
